tests/integration/components/phone-input-test.js

```diff
@@ -1,6 +1,6 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { fillIn, render, find } from '@ember/test-helpers';
+import { fillIn, render, find, typeIn } from '@ember/test-helpers';
```

Comment:
We could clean up to remove `fillIn` usage for `typeIn`

Reply:
(this would be out of scope of this PR)

Reply:
hey @dcyriller sure! I'll open another PR :) I agree with you